### ISSUE TITLE
Update for lucid-2.5

### ISSUE THREAD

**Issue by mietek** [chrisdone/tryhaskell#29]
Attempting to build with `lucid-2.5` instead of `lucid-2.0` results in the following failure:
```
src/TryHaskell.hs:366:61:
   No instance for (Monoid a1) arising from a use of ‘mempty’
   The type variable ‘a1’ is ambiguous
   Note: there are several potential instances:
     instance Monoid a => Monoid (Control.Applicative.Const a b)
       -- Defined in ‘Control.Applicative’
     instance Monoid () -- Defined in ‘Data.Monoid’
     instance (Monoid a, Monoid b) => Monoid (a, b)
       -- Defined in ‘Data.Monoid’
     ...plus 39 others
   In the second argument of ‘p_’, namely ‘mempty’
   In the first argument of ‘td_’, namely
     ‘(p_ [class_ "haskell-icon"] mempty)’
   In a stmt of a 'do' block: td_ (p_ [class_ "haskell-icon"] mempty)

src/TryHaskell.hs:386:44:
   No instance for (Monoid a0) arising from a use of ‘mempty’
   The type variable ‘a0’ is ambiguous
   Note: there are several potential instances:
     instance Monoid a => Monoid (Control.Applicative.Const a b)
       -- Defined in ‘Control.Applicative’
     instance Monoid () -- Defined in ‘Data.Monoid’
     instance (Monoid a, Monoid b) => Monoid (a, b)
       -- Defined in ‘Data.Monoid’
     ...plus 39 others
   In the second argument of ‘span6_’, namely ‘mempty’
   In a stmt of a 'do' block: span6_ [id_ "console"] mempty
   In the first argument of ‘row_’, namely
     ‘(do { span6_ [id_ "console"] mempty;
            span6_ [id_ "guide"] mempty })’
```

**Comment by chrisdone** [comment]
There's a `cabal.config` for GHC 7.8.3, I haven't tested on other version sets yet.

**Comment by chrisdone** [comment]
Okay, I bumped Lucid to 2.6 to fix this.

**Comment by chrisdone** [comment]
Fixed in 07fb75d44a08973a0b458588636aff8a61a06f15.
